services/oc-opencor-base/metadata/metadata.yml

```diff
@@ -0,0 +1,36 @@
+name: opencor
+key: simcore/services/comp/opencor
+type: computational
+integration-version: 1.0.0
+version: 1.0.0
+description: OpenCOR is a cross-platform modelling environment, which can be used to organise, edit, simulate and analyze CellML and SED-ML files
+contact: zhuang@itis.swiss
+authors:
+  - name: Katie Zhuang
+    email: zhuang@itis.swiss
+    affiliation: IT'IS Foundation
+inputs:
+  model_url:
+    displayOrder: 1
+    label: model url
+    description: the URL of a CellML / SED-ML file you would like to run
+    type: string
+    defaultValue: https://models.physiomeproject.org/workspace/5f5/rawfile/125f548ce204c1d815298d2c8c1d9b774d89e3a7/mcintyre_richardson_grill_model_2001.sedml
+  config_file:
+    displayOrder: 2
+    label: configuration file (JSON)
+    description: Used to configure the simulation (using the information in the simulation section, if present), customise the initial state of the model (using the information in the parameters section, if present).
```

Comment:
For consistency, should the description with an uppercase or a lowercase letter? (See your description for `model_url`.)

Reply:
agreed, this is changed to be consistent

Reply:
Actually, the config file can also be used to specify the data that is to be outputted (see https://github.com/opencor/docker).

Reply:
added